src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/gui/ProxyControlGui.java

```diff
@@ -442,6 +483,12 @@ public void actionPerformed(ActionEvent action) {
             }
         } else if (command.equals(ACTION_RESTART)) {
             model.stopProxy();
+            // what is the last number for numbering samplers, ex : 96
+            int iRequestNumber = model.getRequestNumberFromSamplerCreator();
```

Comment:
If the *`i`* at the beginning of `iRequestNumber` is used to indicate an integer, I would remove it and name the variable `requestNumber`. That is the way variables are named in JMeter most of the time. Most developers around JMeter will probably use an IDE, which shows the type of the variable anyway.

Reply:
I use prefix  i for Integer or s for String for local variables but i could change to requestNumber